aten/src/ATen/native/mps/operations/Activation.mm

```diff
@@ -417,6 +417,9 @@ Tensor relu_mps(const Tensor& self) {
   using CachedGraph = MPSUnaryCachedGraph;
   TORCH_CHECK(output.is_mps());
 
+  if(output.numel() == 0)
+    return;
+
```

Comment:
nit: per PyTorch coding standard we should use curly braces even for single line return blocks
```suggestion
  if(output.numel() == 0) {
    return;
  }

```